src/renderer/screens/exchange/Swap2/Form/FormSelectors/ToRow.js

```diff
@@ -46,8 +68,7 @@ export default function ToRow({ toCurrency, setToCurrency, toAmount, setToAmount
             placeholder="0"
             textAlign="right"
             containerProps={amountInputContainerProps}
-            // $FlowFixMe
-            unit={unit}
+            defaultUnit={toCurrency?.units[0]}
```

Comment:
Better use the `getAccountUnit` helper, in case things change

Reply:
Got it, I'm going to

Reply:
As I'm dealing with currencies here (and not accounts), are we agree that when the currency is set by the user from the select component, I have to derivate the related account from the selected currency and save it to the form state?

Once the account is saved, I'll be able to:
- Dealing with units here as I have the account object associated with the currency
- Dynamically render target account row (the last one) from the form based on the account object

Do I understand correctly?

Reply:
That's it, to me, the selection of the `toCurrency` merely does an implicit `toAccount` selection.
So, when I select... BTC I would find the first BTC account as the `toAccount`, and then allow the user to change from that account to another BTC account. If I change the `toCurrency` to ETH, I would find the first ETH account and set it as the `toAccount`

The only added complication is if we select a token currency, I would need to set the `toAccount` and the `toParentAccount` (whatever names you choose, but that's the gist of it)

Reply:
Done.